### ISSUE TITLE
Fullscreen swiping (bottom screeen border) not working on Niagara Home Screen

### ISSUE THREAD

**Issue by oslogrolls** [8bitPit/Niagara-Issues#2437]
### Describe the bug

As a new user of Niagara Pro, I see many workflow benefits, but also a very annoying bug:

My realme 12 Pro on Android 13 is set to gesture navigation and allows you to quickly switch between apps using a left/right swipe directly at the bottom of the screen.

Unfortunately, the Niagara home screen is the only screen that does not support this swipe movement. The screen change does not work even if the search is deactivated (Niagara Settings). However, as soon as the search is deactivated, you can at least switch between the open apps by swiping up from the lower edge of the screen. This gesture change creates inconsistency, which greatly affects the UX, especially during activities such as driving.

Since I can't think of a Niagara event that would conflict with the use of this gesture, I'm assuming it's a bug or an oversight.

Unfortunately, the unsupported swipe event causes me to use Niagara to search for the app even though I know it's already open - just to give focus to its window.

### Steps to reproduce

1. Turn on Gesture Navigation
2. Open a few apps with Nigara set as Launcher. Say one of them is Signal and a relevant discussion is active.
3. Turn off the Screen / let the device auto-lock.
4. Unlock the screen to write a Signal message in the active channel. 
5. The phone may wake up to the Niagara home screen. 
6. There's no fast way to swipe to the already open Signal app.

### Does the issue also occur with other 3rd-party launchers?

Problem doesn't occur with other launchers

### Other 3rd-party launcher(s) tested

realme ui 4.0

### Device name

realme 12 pro

### Android version

Android 13

### Screenshots / Screen recordings

![2023-11-20_11h30_30](https://github.com/8bitPit/Niagara-Issues/assets/48279537/0cec9165-9a62-45e8-84b5-cf52349ebafc)

**Comment by godoyLeonardo** [comment]
I'd like to add that the swipe up navigation gesture, that I use a lot for going into my recent apps, feels wrong when using the Launcher.

I fix for this would be to us to be able to deactivate the "swipe up for favorite apps" gesture that's working right now for short and long swipes.

**Comment by oslogrolls** [comment]
@godoyLeonardo Please open a separate thread (and remove your post above). The Niagara makers want us to raise one issue at a time – and your report is unrelated to mine. I report horizontal swiping not working properly, I have no issues with vertical sliding.

**Comment by godoyLeonardo** [comment]
@oslogrolls  No. Coding wise, my problem is totally inside of the same scope as yours.
